DigitalLearningSolutions.Data.Tests/DataServices/GroupsDataServiceTests.cs

```diff
@@ -211,6 +215,56 @@ public void GetRelatedProgressIdForGroupDelegate_returns_null_with_no_record()
             result.Should().BeNull();
         }
 
+        [Test]
+        public void UpdateGroupDescription_updates_record()
+        {
+            using var transaction = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+            try
+            {
+                // Given
+                const int centerId = 101;
+                const int groupId = 5;
+                const string expectedDescription = "Test group description1";
+
+                // When
+                var isUpdated = groupsDataService.TryUpdateGroupDescription(groupId, centerId, expectedDescription);
+
+                // Then
+                isUpdated.Should().Be(true);

```

Comment:
There are different methods BeTrue and BeFalse. They do the same thing as Be(true)/Be(false), but can you use those instead for this and the next test